config-provisioning/src/main/java/com/yahoo/config/provision/SystemName.java

```diff
@@ -31,17 +34,28 @@ public static SystemName defaultSystem() {
     }
 
     public static SystemName from(String value) {
-        switch (value) {
+        switch (value.toLowerCase()) {
             case "dev": return dev;
             case "cd": return cd;
             case "main": return main;
-            case "public":
-            case "Public": return Public;
+            case "public": return Public;
+            case "publiccd": return PublicCd;
             case "vaas": return vaas;
             default: throw new IllegalArgumentException(String.format("'%s' is not a valid system", value));
         }
     }
 
+    public String value() {
+        switch (this) {
```

Comment:
name().toLowerCase()?

Reply:
I had it that way initially, but decided to follow the pattern in `Environment`: https://github.com/vespa-engine/vespa/blob/01ac4cf6c88718df85ff6d3699920ef60487928d/config-provisioning/src/main/java/com/yahoo/config/provision/Environment.java#L55

Not sure its done this way, in that case its even simpler as they can just call `name()` directly... Might change in next PR.